src/Apps/Artwork/Components/ArtworkDetails/ArtworkDetailsAdditionalInfo.tsx

```diff
@@ -20,6 +20,7 @@ import { ArtworkDefinitionList } from "Apps/Artwork/Components/ArtworkDefinition
 import { useTracking } from "react-tracking"
 import { useArtworkDimensions } from "Apps/Artwork/useArtworkDimensions"
 import { ArtworkSidebarClassificationsModalQueryRenderer } from "Apps/Artwork/Components/ArtworkSidebarClassificationsModal"
+import { ConditionInfoModal } from "Apps/Auction/Routes/Bid/Components/ConditionInfoModal"
```

Comment:
We'll want to move this component into the same folder as `ArtworkDetailsAdditionalInfo` (`src/Apps/Artwork/Components/ArtworkDetails`)